docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -63,6 +63,8 @@ The main way is with an `InputAdornment`.
 This can be used to add a prefix, a suffix, or an action to an input.
 For instance, you can use an icon button to hide or reveal the password.
 
+When using a start adornment, you can provide `startAdornmentWidth` to the `FormControl`. Otherwise, the label will always be shrunk.
```

Comment:
Could we have a dedicated demo for this? `pages/components/text-fields/InputAdornments.js` might starts to be too large, making it hard for the developers to find what they are looking for.

Reply:
@oliviertassinari Done.